src/components/displayAll.js

```diff
@@ -0,0 +1,54 @@
+import React, { useEffect } from 'react';
+import { useNavigate } from 'react-router-dom';
+import { useSelector, useDispatch } from 'react-redux';
+
+import { FetchStockData, ShowStockData } from '../redux/Reducer';
+
+function DisplayAll() {
+  const Stocks = useSelector((state) => state.stocks);
+  const dispatch = useDispatch();
+
+  const Navigate = useNavigate();
+  useEffect(() => {
+    dispatch(FetchStockData());
+  }, []);
+
+  const obtainDetails = (symbol) => {
+    dispatch(ShowStockData(symbol));
+  };
+
+  return (
+    <>
+
+      <h3 className="main-heading"> Crypto currency market data</h3>
+
+      <div className="information">
+
+        {
+
+Stocks.length > 1
+  ? Stocks.map((stock) => (
+    <div key={stock.symbol} className="columnDiv">
+      <button
+        className=""
```

Comment:
- [ ] it would be better if you use the right font given in the design in order to match the design
- the font was given in the design 
<img width="807" alt="Screenshot 2022-11-30 at 17 12 25" src="https://user-images.githubusercontent.com/61790587/205036984-941c9307-10ff-4c7e-aa36-a82109c9aa5f.png">